src/ResXManager.View/Visuals/TranslationItem.cs

```diff
@@ -44,14 +44,25 @@ public TranslationItem(ResourceTableEntry entry, string source, CultureKey targe
             set => _translation = value;
         }
 
-        public bool Apply(string? prefix)
+        public bool UpdateValue(string? prefix)
         {
             if (!_entry.CanEdit(TargetCulture))
                 return false;
 
             return _entry.Values.SetValue(TargetCulture, prefix + Translation);
         }
 
+        public bool UpdateComment(string? prefix)
+        {
+            var neutralLanguageComment = _entry.Comment;

```

Comment:
I think this needs a check for .CanEdit, too, like above

Reply:
I will try to refactor these methods back into one Apply method, as you suggested